src/main/resources/templates/blog.html

```diff
@@ -29,12 +26,9 @@ <h5 class="card-title" th:text="${ blog.title }"></h5>
           <a href="#" class="btn btn-danger">Delete</a>
         </div>
       </div>
-
     </main>
   </div>
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure